Assets/sass/main.scss

```diff
@@ -0,0 +1,398 @@
+$primary: #ec5242;
+$dark: #272a31;
+$third: #d3d3d3;
+$lato: "Lato", sans-serif;
+$coco: "cocogoose", sans-serif;
+
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap");
+
+@font-face {
+  font-family: "cocogoose";
+  src: url(../fonts/Cocogoose.ttf);
+}
+
+html {
+  font-size: 100%;
+}
+
+body {
+  min-width: 320px;
+}
+
+footer {
+  padding: 5.95vh 8.945vw;
+
+  i {
+    font-size: 1.33rem;
+    margin: 5.95vh 0.894vw;
+  }
+}
+
+header {
+  background-image: url(../img/supercars.gif), linear-gradient(#d3d3d3, #d3d3d3fb);
+  background-blend-mode: overlay;
+  background-size: cover;
+  background-position: right;
+  //height: 690px;
+  padding-bottom: 14.89vh;
+}
+
+.navbar-toggler {
+  border: none;
+  font-size: 2rem;
+}
+
+.main-heading {
+  h1 {
+    font-family: $coco;
+    font-size: 2.8rem;
+    padding: 1.48vh 1.789vw;
+    margin-bottom: 1.48vh;
+    letter-spacing: 0.25rem;
+    color: $primary;
+  }
+
+  h2 {
+    font-size: 1.75rem;
+    margin: 1.48vh 1.789vw;
+    color: $primary;
+  }
+
+  p {
+    font-family: $lato;
+    font-size: 1.08rem;
+    margin: 2.23vh 2.683vw;
+    border: 2px solid white;
+    padding: 2.97vh 3.578vw;
+    color: $dark;
+  }
+
+  h3 {
+    font-size: 2rem;
+    font-family: $coco;
+    color: $dark;
+  }
+
+  h4 {
+    color: $dark;
+  }
+}
+
+.main-program {
+  color: $third;
+  background-image: url(../img/blackpattern.jpg), linear-gradient(#d3d3d336, #d3d3d336);
+  background-blend-mode: overlay;
+  background-size: cover;
+
+  button {
+    background-color: $primary;
+    border: none;
+    color: $third;
+    font-size: 1.5rem;
+    padding: 2.97vh 19.678vw;
+    margin: 8.92vh 10.733vw;
+  }
+}
+
+.heading {
+  margin-bottom: 4.46vh;
+
+  h2 {
+    font-family: $lato;
+    font-size: 1.83rem;
+    margin-top: 7.44vh;
+    margin-bottom: 2.98vh;
+  }
+
+  span {
+    width: 8.945vw;
+    height: 0.3vh;
+    background-color: #ec5242;
+  }
+}
+
+.activity {
+  background-color: #d3d3d323;
+  margin: 2.97vh 3.578vw;
+  padding: 2.97vh 3.578vw;
+
+  i {
+    font-size: 2.08rem;
+  }
+
+  h5 {
+    font-size: 1.5rem;
+    color: $primary;
+  }
+
+  p {
+    font-size: 1.08rem;
+    margin: 0;
+    padding-left: 1.789vw;
+  }
+}
+
+.pilots {
+  background-color: white;
+  padding-bottom: 7.44vh;
+}
+
+.pilot {
+  margin: 1.48vh 1.789vw;
+  background-size: 7.44vh 8.945vw;
+  background-repeat: no-repeat;
+  background-position: top left;
+
+  img {
+    margin: 1.48vh 1.789vw;
+  }
+}
+
+.pilot-info {
+  margin: 1.48vh 1.789vw;
+
+  hr {
+    width: 3.578vw;
+  }
+}
+
+.partners {
+  color: $third;
+  background-color: $dark;
+  padding-bottom: 7.44vh;
+}
+
+.partners-logos {
+  padding: 1.48vh 8.945vw;
+}
+
+/* Media Query */
+@media (min-width: 1px) {
```

Comment:
- [ ] (Ok, this is a hard one) Please, the CSS styling is built as **mobile-first**, media queries are used to change the design from mobile to desktop, not the other way around. In this case, you're using the `@media` for mobile, the default style should be for mobile and the `@media` should be used for desktop, I kindly invite you to fix this. I'm sure you'll find a way to solve it as well as possible. 🦁